File: pset6/mario.py
def printSpaces(n):
    for i in range(n):
        print(" ", end="");

while True:
    n = int(input("Height: "))
    if n>-1 and n<24:
        break

spaces=n-1

for i in range(n):
    printSpaces(spaces)
    spaces -= 1
    for j in range(i+1):
        print("#", end="")
    print("  ", end="")
    for j in range(i+1):
        print("#", end="")
    print("")

File: pset1/Rawnix-cs50-2017-x-mario-less-20171215T181347Z/mario.c
#include<stdio.h>
#include<cs50.h>

void printSpaces(int n)
{
    for(int i=0; i<n; i++)
    {
        printf(" ");
    }
}

int main(void)
{
    int h;
    do
    {
        printf("Height: ");
        h = get_int();
    }while(h<0 || h>23);
    int spaces = h-1;
    for(int i=0; i<h; i++)
    {
        printSpaces(spaces);
        spaces--;
        for(int j=0; j<i+2; j++)
        {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}